libs/components/config/src/lib/config-remote-module-reference.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Information about a referenced remote module.
+ */
+export interface SkyuxConfigRemoteModuleReference {
+  /**
+   * The name of the SKY UX SPA containing the remote module.
+   */
+  app: string;
+
+  /**
+   * The path to the remote module.
+   */
+  exposedModule: string;
```

Comment:
What does this represent? Couldn't a container expose any number of modules or components?

Reply:
Yes, it can expose more than one module. Referencing multiple modules in the same application would be the same as referencing multiple routes in the same application:

```
"remoteModules": {
  "referenced": [
    {
      "app": "foo",
      "exposedModule": "./SomeComponent"
    },
    {
      "app": "foo",
      "exposedModule": "./SomeOtherComponent"
    }
  ]
}
```

Reply:
Updated to use this format for remote modules:

```
"remoteModules": {
  "referenced": [
    {
      "app": "foo",
      "exposedModules": ["./SomeComponent", "./SomeOtherComponent"],
      "version": 2
    }
  ]
}
```